app/views/admin/split_contexts/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="form-container sc-m-t--l"%>
+  <%= simple_form_for @split, url: admin_split_context_path, method: :put do |f| %>
+    <%= f.error :base %>
+    <%= f.input :hypothesis, hint: "What do we expect to change with the treatment condition, and why?" %>
+    <%= f.input :context, hint: "When do we assign users to this split? Is the split population constrained to a particular type of user or action? Assignments should happen as \"close\" to the varied experience as possible, so that only relevant users are included in the analysis." %>
```

Comment:
i left a similar comment below but without all this wonderful reasoning. i was suggesting "description" but it sounds like this is trying to be more specific than that. is there going to be a general description field for a split, or would it be fine to call this description and just give a really good prompt for what should be put in this field?